controllers/model/grafana_resources.go

```diff
@@ -20,7 +20,7 @@ func GetGrafanaConfigMap(cr *grafanav1beta1.Grafana, scheme *runtime.Scheme) *v1
 			Namespace: cr.Namespace,
 		},
 	}
-	controllerutil.SetOwnerReference(cr, config, scheme)
+	controllerutil.SetOwnerReference(cr, config, scheme) //nolint:errcheck
```

Comment:
Not sure if we should ignore these errors everywhere, it currently works but it would also be very strange if we can't set ownership (as long as nothing else is owning the resource it shoulden't be a problem.
But you can actually remove for example a pods owner but I assume that the controller re-adds it directly, so we still need the functionality of setting it during a normal reconciliation.

At the same time the ownership should be set during its initial creation.
Maybe it's better to merge this first and then potentially do a refactoring of that